lib/index.js

```diff
@@ -15,6 +14,13 @@ const MongoDBKeyDescriptionStorage =
   require('./MongoDBKeyDescriptionStorage.js');
 require('bedrock-did-context');
 
+const didIo = require('did-io');
+
+// Config did-io to support did:key and did:v1 drivers
+didIo.use('key', require('did-method-key').driver());
+// TODO: Move v1 'test' mode param to config.
+didIo.use('v1', require('did-veres-one').driver({mode: 'test'}));
```

Comment:
How do things go when we want to use some other V1 mode?

Reply:
@mattcollier good point. I think the mode is there just to be the default when generating new DIDs. For resolving (which is what this is doing), we shouldn't need to specify, driver should route based on the DID itself (and the presence of `test` in it).
Double-checking..

Reply:
Yeah, configuring a single implicit instance of "didIo" like this is problematic across multiple modules. I approved for the near term, but we will need a number of architectural fixes in the future.

Reply:
@mattcollier -- Do we want to merge with this and open an issue to address it later?